### ISSUE TITLE
Can't install with latest React Native version

### ISSUE THREAD

**Issue by valentin-debris** [akvelon/react-native-sms-user-consent#27]
Hi,

I'm new in React-native. I started a project from scratch (with `npx react-native@latest init AwesomeProject`), which setup a project with react-native@0.71.8 and react@18.2.0.

When I try to install your module with NPM, the react version is in conflict with the restriction you have (react@^16.8.1).

Do you plan to make it compatible with the latest version ?

Regards,
Valentin

Full stack error :
```
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"18.2.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.1" from @eabdullazyanov/react-native-sms-user-consent@1.0.11
npm ERR! node_modules/@eabdullazyanov/react-native-sms-user-consent
npm ERR!   @eabdullazyanov/react-native-sms-user-consent@"*" from the root project
```

**Comment by shubham0804** [comment]
Have you tried installing the package with the legacy-peer-deps flag?

**Comment by mrshakirov** [comment]
Please check out your issue on the new published version 1.1.0, should be fixed (:
